recipes/variant-effect-predictor/build.sh

```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```

Comment:
true, but that doesn't really relate to some version does it? This just fixes to plugin to some arbitrary point in the history of the repo. How would we pinpoint the right commit?